COUNTER_Robots_list.json

```diff
@@ -72,6 +72,10 @@
     "pattern": "almaden",
     "last_changed": "2017-08-08"
   },
+  {
+    "pattern": "AntBot",
```

Comment:
This would be matched by the generic `bot` pattern (and remember, this list is supposed to be use with case insensitivity).